### PR TITLE
Fix lstrip used on nil variable

### PR DESCRIPTION
**Pull request by ghost**
the @text var is not defined with this profile
joindiaspora.com/public/kostasvoulazeris.atom

### CODE REVIEW

**Comment by Flaburgan** [comment]
There is also https://github.com/diaspora/diaspora/pull/7420 to solve https://github.com/diaspora/diaspora/issues/7249 I don't know what's the best approach.

File: lib/diaspora/message_renderer.rb [hunk 1]
```diff
@@ -210,6 +210,8 @@ def markdownified opts={}
     # @option opts [Integer] :length (70) Truncate the title to
     #   this length. If not given defaults to 70.
     def title opts={}
+      @text = "" if @text.nil?
+
       # Setext-style header
       heading = if /\A(?<setext_content>.{1,200})\n(?:={1,200}|-{1,200})(?:\r?\n|$)/ =~ @text.lstrip
```
Comment (cmrd-senya) [inline lib/diaspora/message_renderer.rb]: I would write
```ruby
@text.try(:lstrip)
```
instead but that's just a question of style

**Comment by jhass** [comment]
I think I quite prefer Steffen's approach :) The message render shouldn't handle nil itself but fail hard on it IMO, since swallowing it at this point makes finding the origin only harder.

**Comment by cmrd-senya** [comment]
> I think I quite prefer Steffen's approach :) The message render shouldn't handle nil itself but fail hard on it IMO, since swallowing it at this point makes finding the origin only harder.

Then it's probably better to have a check there and raise an exception, so that it is explicit that the `title` call was made improperly with `@text == nil`.

**Comment by Zauberstuhl** [comment]
ups haven't searched for it. just hot-fixed it on JD and wanted to show you guys that this could be a problem. thanks anyway

**Pull request closed without merging**